modules/lsp/src/main/scala/playground/lsp/ServerLoader.scala

```diff
@@ -93,8 +93,11 @@ object ServerLoader {
                 stateRef.get.flatMap { state =>
                   BuildLoader[F]
                     .load(workspaceFolders)
-                    .map { case params =>
-                      PrepareResult(params, !state.lastUsedConfig.contains(params.config))
+                    .map { params =>
+                      PrepareResult(
+                        params,
+                        !state.lastUsedConfig.map(_.config).contains_(params.config),
```

Comment:
This would actually never have passed, so the whole `PrepareResult` would always contain `isChanged=true`